tests/test_base_env.py

```diff
@@ -19,9 +18,9 @@ def __init__(self):
         """Build environment."""
         nS = 3
         nA = 2
-        transition_matrix = np.random.rand(nS, nA, nS)
+        transition_matrix = np.random.random((nS, nA, nS))
```

Comment:
Probably good to seed this explicitly? Possibly also using `self.np_random` or `self.rand_state`